pages/integrations/sso/azure_ad.md.erb

```diff
@@ -92,8 +92,7 @@ Then go to your [Azure Admin Console](https://portal.azure.com/) and select the
     1. Toggle _Enabled_ to _No_ and click _Save_
 1. Customise the User mappings:
     1. Expand _Mappings_, then click _Provision Azure Active Directory Users_
```

Comment:
Would it make sense to have these as alphabetical under the numbered list? I've found it a bit difficult to parse with them all being numbered.

Reply:
Looks like our markdown parser doesn't do this 🥲

Reply:
I've asked @lachlanhardy to take a quick squiz at this. Agreed that it would be better to have them in a different style.

Reply:
I'll sort it in https://github.com/buildkite/docs/pull/1164